src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -130,7 +141,9 @@ protected void runTool( final JavaSparkContext ctx ) {
 
         // write the output file
         final SAMFileHeader cleanHeader = new SAMFileHeader(header.getSequenceDictionary());
-        AlignedAssemblyOrExcuse.writeSAMFile(outputSAM, cleanHeader, alignedAssemblyOrExcuseList);
+        cleanHeader.setSortOrder(outputSortOrder);
```

Comment:
Could you move this order setting line of code into `AlignedAssemblyOrExcuse.writeSAMFile()`? It is better to ask the function that does the writing to set the order, IMO.